source/classes.tex

```diff
@@ -5782,7 +5782,8 @@
 not the most derived class.
 
 \pnum
-A temporary expression bound to a reference member in a \grammarterm{mem-initializer}
+A program extending the lifetime of a temporary object
+by binding it to a reference member in a \grammarterm{mem-initializer}
 is ill-formed.
```

Comment:
Can we phrase this as a regular "shall not" rule, rather than "A program ... is ill-formed"?

Reply:
Something like: "A program shall not (attempt to?) extend the lifetime of a temporary object by binding it to a reference member ..."?